react/src/components/awards/SprkAward.js

```diff
@@ -78,38 +82,61 @@ const SprkAward = (props) => {
         })}
       </SprkStack>
 
-      <SprkToggle
-        title={disclaimerTitle}
-        analyticsString={disclaimerAnalytics}
-        additionalClasses="sprk-o-Stack__item"
-      >
-        <p className="sprk-b-TypeBodyFour sprk-u-pts sprk-u-pbs">
-          {disclaimerText}
-        </p>
-      </SprkToggle>
+      {disclaimerTitle && disclaimerText && (
+        <SprkToggle
+          title={disclaimerTitle}
+          analyticsString={disclaimerAnalytics}
+          additionalClasses="sprk-o-Stack__item"
+        >
+          <p className="sprk-b-TypeBodyFour sprk-u-pts sprk-u-pbs">
+            {disclaimerText}
+          </p>
+        </SprkToggle>
+      )}
     </SprkStack>
   );
 };
 
 SprkAward.defaultProps = {
   splitAt: 'small',
   itemSpacing: 'medium',
-  additionalClasses: '',
-  idString: '',
-  heading: '',
-  disclaimerText: '',
-  disclaimerAnalytics: '',
-  disclaimerTitle: '',
 };
 
 SprkAward.propTypes = {
-  /** Determines the spacing between the items. */
-  itemSpacing: PropTypes.string,
+  /**
+   * Determines the spacing between the items.
+   * `misc-a`, `misc-b`, etc are deprecated. Use `miscA`, `miscB`, etc
+   * instead.
+   */
+  itemSpacing: PropTypes.oneOf([
+    'tiny',
+    'small',
+    'medium',
+    'large',
+    'huge',
+    'misc-a',
+    'misc-b',
+    'misc-c',
+    'misc-d',
+    'miscA',
+    'miscB',
+    'miscC',
+    'miscD',
+    '',
```

Comment:
Do we need these empty strings on `itemSpacing` and `splitAt`? They have defaults, so there shouldn't be a need to allow for an empty string.